source/scss/fti-bootstrap/_custom-variables.scss

```diff
@@ -40,10 +40,11 @@
 
 $gray-100: #f4f4f4;
 $gray-300: #b3b3b3;
+$gray-400: #929497;
 $gray-600: #666666;
 $gray-800: #424242;
 
-$red: #c01d1d;
+$red: #fc1119;
```

Comment:
We are changing a red. Is this on purpose? Are all previous components ok with it?